api/test/queries.test.js

```diff
@@ -489,19 +489,130 @@ describe('queries', () => {
               query: `{
                  dwellings:dwellingsInFSA(
                   forwardSortationArea: "C1A"
-                  filter: {field: city eq: "Charlottetown"}
+                  filter: {field: dwellingCity eq: "Charlottetown"}
+                 ) {
+                   results {
+                     city
+                   }
+                 }
+               }`,
+            })
+          let { dwellings: { results: [first] } } = response.body.data
+          expect(first.city).toEqual('Charlottetown')
+        })
+      })
+
+      describe('lt: less than', () => {
+        it('filters out results where the selected field has a value less than the selected value', async () => {
+          let response = await request(server)
+            .post('/graphql')
+            .set('Content-Type', 'application/json; charset=utf-8')
+            .send({
+              query: `{
+                 dwellings:dwellingsInFSA(
+                  forwardSortationArea: "C1A"
+                  filter: {field: dwellingYearBuilt lt: "2000"}
+                 ) {
+                     results {
+                       yearBuilt
+                    }
+                 }
+               }`,
+            })
+          let { dwellings: { results: [first] } } = response.body.data
+          expect(first.yearBuilt).toEqual(1900)
+        })
+      })
+
+      describe('eq: equal to', () => {
+        it('filters out results where the selected field has a value equal to the selected value', async () => {
+          let response = await request(server)
+            .post('/graphql')
+            .set('Content-Type', 'application/json; charset=utf-8')
+            .send({
+              query: `{
+                 dwellings:dwellingsInFSA(
+                  forwardSortationArea: "C1A"
+                  filter: {field: dwellingYearBuilt eq: "1900"}
                  ) {
                    results {
                      yearBuilt
                    }
                  }
                }`,
             })
+
```

Comment:
Might be good to have a test in here for not finding something as well. 

Like, if we search `filter: {field: dwellingYearBuilt eq: "1899"}`, then we get no results.

Reply:
Agree. I'll add that shortly.